README.md

```diff
@@ -1,8 +1,35 @@
-# Kurtosis Usage Examples
+# Awesome Kurtosis [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
 
-This is the official Kurtosis examples library.
+<img src="./logo.png" width="500">
 
-Each example resides in a separate folder. We currently offer the following different types of projects:
+> A curated list of Kurtosis Starlark Packages
 
-* Writing a unit test using the Kurtosis Golang SDK
-* Running a Kurtosis Starlark package
+These Kurtosis packages and examples provide a starting point for how to setup and test different environments using a Kurtosis Stalrlark package and
+to manage their lifetime using the Kurtosis CLI.
+
+> **Note**
+> The following samples are intended for use in local development and testing environments. These samples must not be deployed in production environments.
```

Comment:
Why do we say "must not be deployed in production"? Is this a suggestion or is it a Business Source License (BSL) limitation? If the latter, we should be transparent about that.